packages/styles/scss/components/filter-panel/_filter-panel.scss

```diff
@@ -0,0 +1,426 @@
+/**
```

Comment:
This whole file needs basic formatting. There should be line breaks separating blocks. Also, there is an inconsistent use of `_` and `__` (single `_` is never used in our naming conventions) that needs to be cleaned up.